_content/monographs/soto-zen-intro-to-zazen_soto-zen-buddhism-international-center.md

```diff
@@ -0,0 +1,21 @@
+---
+title: "Sōtō Zen: An Introduction to Zazen"
+editor: "Sōtō Zen Buddhism International Center"
+external_url: "https://www.sotozen.com/eng/library/leaflet/sotozen/pdf/soto_zen.pdf"
+drive_links:
+  - "https://drive.google.com/file/d/1FCf9TjCziujXivEaRgqyd8TJQ1I5sGaQ/view?usp=sharing"
+course: japanese
+tags:
+  - zen
+  - japanese-roots
```

Comment:
No need for `japanese-roots` but since this is introductory level we could tag `mahayana` maybe?